applications/tasks/src/kz_tasks_trigger.erl

```diff
@@ -148,7 +141,7 @@ handle_info({'timeout', Ref, _Msg}, #state{day_ref = Ref}=State) ->
     {'noreply', State#state{day_ref = day_timer()}};
 
 handle_info({'timeout', Ref, _Msg}, #state{browse_dbs_ref = Ref}=State) ->
-    _Pid = kz_process:spawn(fun browse_dbs_for_triggers/1, [Ref]),
+    _Pid = kz_process:spawn(fun kt_compactor:browse_dbs_for_triggers/1, [Ref]),
```

Comment:
kt_compactor is a task executor, not a "core" task module and should not be called from within here, no?